lightningd/channel_control.c

```diff
@@ -376,9 +376,9 @@ void peer_start_channeld(struct channel *channel,
 
 	if (channel->scid) {
 		scid = *channel->scid;
-		/* Subtle: depth=1 at funding height. */
-		reached_announce_depth = get_block_height(ld->topology) + 1 >=
-				       short_channel_id_blocknum(&scid) + ANNOUNCE_MIN_DEPTH;
+		reached_announce_depth
+			= is_scid_depth_announcable(&scid,
```

Comment:
Why not using directly the above pointer instead of a temporary variable ?

Reply:
Minimal change, really: the replaced code used &scid too.